### PR TITLE
[ODLA TRT] add submodule Tensorrt-7.0.0 and odla trt plugin files

### PR DESCRIPTION
**Pull request by shuhand**

### CODE REVIEW

File: ODLA/platforms/tensorrt/plugins/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,163 @@
+# ==============================================================================
+# Copyright (C) 2020 Alibaba Group Holding Limited.
```
Comment (weimingzha0) [inline ODLA/platforms/tensorrt/plugins/CMakeLists.txt:2]: 2021?

File: ODLA/platforms/tensorrt/plugins/initPlugin.h [hunk 2]
```diff
@@ -0,0 +1,26 @@
+//===- initPlugin.h -------------------------------------------------------===//
+//
+// Copyright (C) 2019-2020 Alibaba Group Holding Limited.
```
Comment (weimingzha0) [inline ODLA/platforms/tensorrt/plugins/initPlugin.h:3]: 2021?

File: .github/workflows/main.yaml [hunk 3]
```diff
@@ -16,13 +16,15 @@ jobs:
         uses: actions/checkout@v2
         with:
           path: halo
-          submodules: true
-
+      - name: Checkout submodule
+        run: |
+             git submodule sync --recursive
```
Comment (weimingzha0) [inline .github/workflows/main.yaml:21]: how about just use "submodule: recursive" ?

Comment (shuhand) [reply .github/workflows/main.yaml]: resolved

**Review by weimingzha0 — APPROVED** [review]

**Pull request merged**